mongo/integration/unified/client_entity.go

```diff
@@ -160,20 +160,36 @@ func (c *clientEntity) stopListeningForEvents() {
 	c.setRecordEvents(false)
 }
 
-func (c *clientEntity) isIgnoredEvent(event *event.CommandStartedEvent) bool {
+func (c *clientEntity) isIgnoredStartedEvent(event *event.CommandStartedEvent) bool {
```

Comment:
The logic in this function is basically the same as in `isIgnoredSucceededCommand`. Can we add a new `isIgnoredEvent(commandName string, document bson.Raw)` helper and pass it either `event.Command` or `event.Reply` from one of these functions?

Reply:
Yep! Used `isIgnoredEvent` for both.